src/components/modal/_modal.scss

```diff
@@ -195,14 +195,14 @@
     max-height: 100%;
     height: 100%;
 
-    @include breakpoint(md) {
+    @include carbon--breakpoint(md) {
```

Comment:
Would it be helpful on the carbon side to create aliases for breakpoint and toggle based on feature flags? I don't know if it'd make it easier to work with the mixins without the namespace or not so wanted to offer a suggestion 😄

Reply:
🤔 I think using prefixed one is fine, but open for people's thoughts. BTW feature flag for `v9`/`v10` shouldn't be around for long period of time from now.